gift-certificate-application/model/src/main/java/com/epam/esm/listener/GiftCertificateListener.java

```diff
@@ -0,0 +1,24 @@
+package com.epam.esm.listener;
+
+import com.epam.esm.entity.GiftCertificate;
+
+import javax.persistence.PrePersist;
+import javax.persistence.PreUpdate;
+import java.time.LocalDateTime;
+
+public class GiftCertificateListener {
+    private LocalDateTime currentDateTime;
```

Comment:
For what reason we need a class variable here? You can create a local variable in the methods instead